compat/src/internal.d.ts

```diff
@@ -16,7 +16,8 @@ export interface Component<P = {}, S = {}> extends PreactComponent<P, S> {
 
 	_childDidSuspend?(
 		error: Promise<void>,
-		suspendingComponent: Component<any, any>
+		suspendingComponent: Component<any, any>,
+		oldVNode?: VNode
```

Comment:
I think we can remove this - I didn't see anywhere where an implementation of _childDidSuspend took a third param

Reply:
I wasn't sure if we needed to add it in order to fix the issue you mentioned about picking the wrong vnode on which to check `_hydrating`.